application/common/config/main.php

```diff
@@ -24,11 +25,26 @@
 
 $mfaNumBackupCodes = Env::get('MFA_NUM_BACKUPCODES', 10);
 
+$envValidator = function($name, $key, $config) {
```

Comment:
Would you mind switching to simply declaring the function, rather than assigning the function to a variable?

```suggestion
function assertEnvVarIsSet($name, $key, $config) {
```

I think that will be easier to follow. In PHP, it's still (relatively) rare to see a function assigned to a variable (unlike in JavaScript). It's supported, just a little unexpected, and in this case it's not necessary to do so.

Reply:
(Just as a record) It looks like it can't use a normal function, since the file must be required by multiple other files, which makes it look like the function is being re-defined.